holodeck/environments.py

```diff
@@ -212,9 +212,23 @@ def teleport(self, agent_name, location=None, rotation=None):
             rotation (np.ndarray or list): A new rotation target for the agent.
                 If no rotation is given, it isn't rotated, but may still be teleported. Defaults to None.
         """
-        self.agents[agent_name].teleport(location * 100, rotation)  # * 100 to convert m to cm
+        self.agents[agent_name].teleport(np.array(location) * 100, np.array(rotation))  # * 100 to convert m to cm
```

Comment:
I think we should fix these conversions on the C++ side. Maybe make an issue for now, and someone should go through and check that every function expects SI units.